src/content/overview/fleet management/_index.md

```diff
@@ -6,7 +6,22 @@ menu:
   principal:
     parent: overview
     identifier: overview-fleet-management
-last_review_date: 2024-03-18
+last_review_date: 2024-04-18
 owner:
   - https://github.com/orgs/giantswarm/teams/sig-product
 ---
+
+Our experience tells us that managing a large microservices platform is not easy. Ideally, workloads are created ready to run in the cloud and are optimized for the container lifecycle. But even so, when you need to manage several providers, regions and environments, the complexity multiplies exponentially. Having that in mind, we have created a some abstractions to help you dealing with that complexity in a more efficient way.
+
+## Features
+
+The fleet management features that our product offers are:
+
+- **Multi-environment**: We support multiple providers, regions and environments. Platform engineers can map their company structure to our platform structure in order to ease the management of their teams and workloads.
+- **Configuration management**: We rely on GitOps principles to manage the configuration of your clusters, environments and workloads having multiple layers of configuration.
```

Comment:
```suggestion
- **Configuration management**: Manage configuration of your clusters, environments and workloads with GitOps, a versioned, single source of truth repository that shows the currently deployed state.
```

I've removed the "multiple layers" part since it can sound like complexity. Maybe you have an idea to instead explain the benefit in one sentence? Such as: making environments consistent, yet flexibly updateable (I didn't find a good sentence here).